src/ImageSharp/Common/Helpers/ImageMaths.cs

```diff
@@ -128,6 +130,15 @@ public static int ModuloP2(int x, int m)
             return x & (m - 1);
         }
 
+        /// <summary>
+        /// Converts degrees to radians
+        /// </summary>
+        [MethodImpl(InliningOptions.ShortMethod)]
+        public static float DegreesToRadians(float degrees)
```

Comment:
`MathFExtensions.DegreeToRadian` in Core provides this already.

Reply:
It seemed internal to me.

Reply:
we use the entire MathF internal class on unsupported target frameworks. Dunno what rules we should apply.

Reply:
`MathF` might be an exception because it's API is fixed, so we can't break compatibility by accident.

Reply:
I've updated Core to expose new `GeometryUtilities`